### DISCUSSION TITLE
Static HTML site links not working

### DISCUSSION THREAD

**Discussion by derekbender** [community/community#46861] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm in the process of moving my static HTML site into Github Pages. I created the repo and uploaded the static files and images as I had them previously:

- index.html
- page1.html
- images (folder)
    - image1.png
    - image2.png
- css (folder)
    - css1.css
    - css2.css
  
The index file works fine (user.github.io/index/) but I'm getting a 404 when try to click the link for page1.html. Also if I navigate directly to page1 (user.github.io/index/page1.html), the images won't load.

**Comment by radar-blip** [comment, marked as answer]
I dont see a page1.html in your repository. I see index2.html (which works).

Also, some of your images have spaces included (not allowed). This should be fixed in both the image and the html. 👍🏻 

So just change `ids 2.png` to `ids2.png`
that should fix it.

**Reply by derekbender** [reply]
Sorry, the list above was just an example. The page in question in instacart.html which is linked from index.html.

**Reply by radar-blip** [reply]
Okay. Try this:

Change this link (in index.html): `<li style="margin-bottom: 0rem;"><a href="../instacart.html">View some of my work from Instacart</a></li>`

to this: `<li style="margin-bottom: 0rem;"><a href="instacart.html">View some of my work from Instacart</a></li>`


And, this link to your image: `<img class="c-media" src="../images/opener.png">`
change to: `<img class="c-media" src="images/opener.png">`

**Reply by derekbender** [reply]
It works, thanks!

**Reply by radar-blip** [reply]
Awesome! & You're welcome!!
